contrib/charts/cert-manager/templates/NOTES.txt

```diff
@@ -1,15 +1,58 @@
-cert-manager has been deployed successfully!
+{{- if .Release.IsInstall }}
+cert-manager and it's CRDs have been installed successfully!
```

Comment:
Drop the "and its CRDs" bit here. cert-manager is the unit of deployment for most our end users, CRDs are an implementation detail